decidim-core/lib/decidim/gamification/badge.rb

```diff
@@ -27,7 +27,9 @@ class Badge
       attribute :reset, Proc
 
       validates :name, :levels, presence: true
-      validates :levels, empty: false
+      # I do not know what kind of validation is this. In rails 6.0.3 the only validation is
```

Comment:
I cannot find it either in rails guides 🤷🏽‍♂️ 
Specs are passing with this commented out, but we should keep researching a little if this is deprecated or what's happening here

Reply:
I could not find any reference of this validator in the Rails core since version 4.2 .
Considering is still working, we should move forward.

Reply:
Let's remove the comments, then.

Reply:
just pushed the fix for this.